### PR TITLE
README.md & CONTRIBUTE.md update

### PR DESCRIPTION
**Pull request by jansenbob**
**WARNING: Should only be merged when https://github.com/objectiv/objectiv.io/pull/157 is live, otherwise it will link to an outdated version of the collector guide.**

- Fix inconsistencies in caps for open taxonomy mentions
- Add link to updated collector guide
- Update readme intro to better represent core concepts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
 ![Objectiv Logo](https://objectiv.io/docs/img/logo-objectiv-large.svg "Objectiv Logo")
 
-Objectiv is a data collection & modeling library that puts the data scientist first. It is built around 
-[the open taxonomy of analytics](https://objectiv.io/docs/taxonomy), which is our proposal for a common way to collect, 
-structure and validate data. With Objectiv, you create a 
-[contextual layer for your application](https://objectiv.io/docs/tracking/core-concepts/tagging) by mapping it to the taxonomy, 
-with the goal of collecting better data and more effective modeling.
+Objectiv is a data collection & modeling library that puts the data scientist first. 
+
+It proposes the adoption of [a common taxonomy for analytics](https://www.objectiv.io/taxonomy), enabling a shared way to collect well-structured, pre-validated data that is ready to model on without significant gruntwork. Models & datasets embracing the taxonomy are reusable and interchangeable, allowing data scientists to build on knowledge and practises of others.
```
Comment (borft) [inline README.md:5]: shouldn't it be practices, as in practice, but plural, rather than the practises the verb

Comment (jansenbob) [reply README.md:5]: I'm not sure what you're suggesting. 

"allowing data scientists to build on knowledge and _the_ practises of others."?

Comment (borft) [reply README.md:5]: practises is a verb, I think you need a noun, which is spelled practices

Comment (jansenbob) [reply README.md:5]: duh i reread it 3 times and completely missed the c instead of s. gotcha, yes it should. will change

**Pull request closed without merging**